scripts/addIngredientController.js

```diff
@@ -1,6 +1,8 @@
-var addIngredientController = {};
+var addIngredientController = {
+
+  show: function(cocktailsToMakeController){
+    myBar.getTemplates();
+    possibleBarIngredients.determineDataLocation(cocktailsToMakeController);
```

Comment:
There's no reason for you to be using WebSQL for some of the storage (barStorage), and local storage for the rest of the storage (localBar, storedIngredientsList). I recommend you to refactor to switch all your storage to a single storage location. This will simplify further improvements / maintenance.